envergo/hedges/services.py

```diff
@@ -65,6 +67,32 @@ def _check_plantation_result_matrix():
 _check_plantation_result_matrix()
 
 
+# This method is outside the PlantationEvaluator class because it makes it
+# easier to patch it in tests.
+def get_replantation_coefficient(moulinette):
+    """Get the "R" value.
+
+    It depends on the activated criteria.
+    """
+    ep_R = D("0")
+    if moulinette.ep.is_activated():
+        ep = moulinette.ep.criteria.all().first()
+        if ep:
+            form = ep.get_settings_form()
+            form.is_valid()
+            ep_R = form.cleaned_data.get("replantation_coefficient", D("0"))
+
+    ep_bcae8 = D("1")
+    if moulinette.conditionnalite_pac.is_activated():
+        bcae8 = moulinette.conditionnalite_pac.criteria.all().first()
+        if bcae8:
+            form = bcae8.get_settings_form()
+            form.is_valid()
+            ep_bcae8 = form.cleaned_data.get("replantation_coefficient", D("1"))
+
+    return float(max(ep_R, ep_bcae8))
```

Comment:
Il y avait pas une histoire ou il ne peut pas etre en dessous de 1 ? 
```suggestion
    return float(max(ep_R, ep_bcae8, D("1")))
```